domain_alias/tests/src/Kernel/DomainAliasDomainDeleteTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Drupal\Tests\domain_alias\Kernel;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\Tests\domain\Functional\DomainTestBase;
+use Drupal\Tests\domain_alias\Functional\DomainAliasTestTrait;
+
+/**
+ * Tests that aliases are deleted on domain delete.
+ *
+ * @group domain_alias
+ */
+class DomainAliasDomainDeleteTest extends DomainTestBase {
+
+  use DomainAliasTestTrait;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = array('domain', 'domain_alias');
```

Comment:
nitpicking: array shorthand?
Will need to submit a PR that runs phpcs with the Drupal 8 standards

Reply:
Yeah, that's a copy paste from another test that we can clean up later.